test/FastTests/Issues/RavenDB_7493.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Linq;
+using System.Reflection;
+using Raven.Client.Json.Converters;
+using Raven.Server.Json;
+using Raven.Server.ServerWide;
+using Raven.Server.Utils;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using Xunit;
+
+namespace FastTests.Issues
+{
+    public class RavenDB_7493 : NoDisposalNeeded
+    {
+        [Fact]
+        public void Ensure_deserialization_routines_are_properly_created()
+        {
+            using (var context = JsonOperationContext.ShortTermSingleUse())
+            {
+                var djv = new DynamicJsonValue();
+
+                var emptyBlittable = context.ReadObject(djv, "goo");
+
+                var ea = new ExceptionAggregator("Could not create deserialization routines");
+
+                foreach (var type in new []{ typeof(JsonDeserializationClient), typeof(JsonDeserializationCluster), typeof(JsonDeserializationServer) })
```

Comment:
@ayende yes, but we can only know that only during execution, and because of that we are doing ```func(emptyBlittable)``` couple lines below.